### PR TITLE
Add task 2

### PR DESCRIPTION
**Pull request by spokk**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -337,626 +318,731 @@
...
+          <li data-target="pane-1" id="1" class="is-active">
+            <a href="#">
+              <span class="icon is-small"><i class="fa fa-image"></i></span>
+              <span class="tab tab-active">Pictures</span>
+            </a>
+          </li>
+          <li data-target="pane-3" id="3">
+            <a href="#">
+              <span class="icon is-small"><i class="fab fa-superpowers"></i></span>
+              <span class="tab ">Team</span>
```
Comment (IrinaShilina) [inline index.html:339]: Bad contrast here

File: index.html [hunk 2]
```diff
@@ -1138,39 +1227,48 @@ <h1 class="title is-size-4">Components</h1>
...
     <div class="modal-content  is-image">
-      <img src="https://source.unsplash.com/random/?sig=3">
+      <img src="https://source.unsplash.com/random/?sig=3" alt="random image">
     </div>
-    <button class="modal-close is-large"></button>
+    <button class="modal-close is-large"><span class="visually-hidden">Close modal</span></button>
   </div>
 
   <script src="https://unpkg.com/bulma-modal-fx/dist/js/modal-fx.min.js"></script>
+  <script src="https://cdn.disability.illinois.edu/skipto.min.js"></script>
```
Comment (IrinaShilina) [inline index.html:1270]: We usually apply `visually-hidden` styles to the "skip" link.

File: index.html [hunk 3]
```diff
@@ -73,71 +77,113 @@
...
+          <span aria-hidden="true">→</span>
+        </li>
+        <li>
+          <a href="#">
+            Parent
+          </a>
+          <span aria-hidden="true">→</span>
+        </li>
+        <li>
+          <a href="#" aria-current="location">
```
Comment (IrinaShilina) [inline index.html:111]: `aria-current="page"` may be more relevant. https://www.digitala11y.com/aria-current-state/

File: index.html [hunk 4]
```diff
@@ -337,626 +318,731 @@
...
-      <form class="column is-half is-offset-one-quarter box">
-        <div class="field">
-          <label class="label">Name</label>
-          <div class="control">
-            <input class="input" type="text" placeholder="Text input">
+        <div class="columns">
+          <div class="column is-3">
+            <button class="button is-primary is-block is-alt is-large">New Post</button>
+            <div class="menu">
+              <ul class="menu-list forum-list">
```
Comment (IrinaShilina) [inline index.html:794]: Looks like items should be links as well as those are the forum's tags.

File: index.html [hunk 5]
```diff
@@ -1064,72 +1150,75 @@ <h1 class="title is-size-4">Components</h1>
...
-          <a><i class="fab fa-facebook fa-2x"></i></a>
-          <a><i class="fab fa-twitter fa-2x"></i></a>
+          <a href="#"><i class="fab fa-github-alt fa-2x"></i><span class="visually-hidden">Github External
+              link</span></a>
+          <a href="#"><i class="fab fa-youtube fa-2x"></i><span class="visually-hidden">Youtube External link</span></a>
+          <a href="#"><i class="fab fa-facebook fa-2x"></i><span class="visually-hidden">Facebook External
+              link</span></a>
+          <a href="#"><i class="fab fa-twitter fa-2x"></i><span class="visually-hidden">Twitter External link</span></a>
         </div>
         <p>The source code is licensed <a href="http://opensource.org/licenses/mit-license.php">MIT</a>.</p>
```
Comment (IrinaShilina) [inline index.html:1203]: According to the task, you should open this link in a new tab and do the following:
image

File: index.html (hunk 1 above)
Comment (spokk) [reply index.html]: Поправил

Screenshot 2021-08-10 230831

File: index.html (hunk 2 above)
Comment (spokk) [reply index.html]: used 'popup' option from documentation 
image

File: index.html (hunk 3 above)
Comment (spokk) [reply index.html]: fixed

File: index.html (hunk 4 above)
Comment (spokk) [reply index.html:794]: From a design, it looks like it should be buttons. Did it as buttons 
image

File: index.html (hunk 5 above)
Comment (spokk) [reply index.html]: Fixed and checked in NVDA.

**Pull request merged**